scheduler.go

```diff
@@ -696,6 +696,17 @@ Jobs:
 	return nil, ErrJobNotFoundWithTag
 }
 
+// Set the first weekday of month
+func (s *Scheduler) EveryMonthFirstWeekday(wd time.Weekday) *Scheduler {
```

Comment:
Let's rename this to:
```suggestion
func (s *Scheduler) MonthFirstWeekday(weekday time.Weekday) *Scheduler {
```